FBSimulatorControl/Management/FBSimulatorPool.m

```diff
@@ -373,7 +382,7 @@ - (BOOL)blanketKillSimulatorsFromDifferentXcodeVersion:(NSError **)error
   // This command kills all Simulator.app binaries that *don't* match the current Simulator Binary Path.
   NSString *simulatorBinaryPath = self.configuration.simulatorApplication.binary.path;
   NSString *command = [NSString stringWithFormat:
-    @"pgrep -lf Simulator.app | grep -v %@ | awk '{print $1}' | xargs kill",
```

Comment:
This string was prepended to [the string in `blanketKillSimulatorAppsPidProducingCommand:`](https://github.com/facebook/FBSimulatorControl/pull/63/files#diff-2a718c5690bba7ddc88dbb1d1c8dec3eR416) so you'd end up with `| xargs kill | xargs kill` otherwise. Verified this with the debugger.